better-font-awesome.php

```diff
@@ -513,18 +543,67 @@ public function text_callback( $args ) {
      * @return  string  Usage text.
      */
     public function get_usage_text() {
```

Comment:
So I mentioned templating. The idea there would be to do something like..

```
public function get_usage_text() { 
   ob_start();
   include __DIR__ . '/templates/usage-text.php'; 
   return ob_get_clean();
}
```

All your output would then be contained within usage-text.php

Makes readability a bit easier for other developers by keeping the more interesting handling separate from html output.